development/Dockerfile

```diff
@@ -22,13 +22,14 @@ ENV prometheus_multiproc_dir=/prom_cache
 ENV NAUTOBOT_ROOT ${NAUTOBOT_ROOT}
 
 # Don't need virtual environments in container, parallelized installer has inconsistent behavior
-RUN poetry config virtualenvs.create false \
+RUN poetry self update \
```

Comment:
Does this bust the cache?

Reply:
ok, is this from cookie cutter? If not, should it be?

Reply:
last commit 5e20c6a is using dev-standards recommendation